controls/V-71997.rb

```diff
@@ -48,9 +28,10 @@
 
 If the release is not supported by the vendor, this is a finding."
   tag "fix": "Upgrade to a supported version of the operating system."
+  tag "fix_id": "F-78349r1_fix"
 
-  # @todo update to match within list
+  # TODO use an array attribute of supported DISTROS and use the be_in matcher?
```

Comment:
I'm not going to block on this, but this should probably be based on the current date and the content of https://access.redhat.com/support/policy/updates/errata#vii